spec/solution_1_spec.rb

```diff
@@ -0,0 +1,79 @@
+require_relative '../solution_1'
+
+describe Tutorial do
+  let(:klass) { described_class.new }
+
+  describe 'solution' do
+    subject { klass.solution(x) }
+
+    context 'return and reverse number' do
+      let(:x) { 123 }
+      it 'when 123 return 321' do
+        expect(subject).to eq '321'
+      end
+    end
+
+    context 'return and reverse number' do
+      let(:x) { 456 }
+      it 'when 456 return 654' do
+        expect(subject).to eq '654'
+      end
```

Comment:
optional: when `it` block has only one expected, we can change to `it { expect(subject).to eq '654' }`